LayoutTests/fast/harness/results.html

```diff
@@ -331,9 +331,9 @@
         let extensionIndex = buffer.lastIndexOf(".");
         let filename = buffer.substr(0, extensionIndex);
         // Test names that are acutally process names are treated like they don't have any extension
-        if (filename.match(/com\.apple\..+/) || filename.match(/org\.WebKit\..+/)) {
+        if (extensionIndex === -1 || filename.match(/com\.apple\..+/) || filename.match(/org\.WebKit\..+/)) {
             return [testName, '', ''];
-        }
```

Comment:
This seems incorrect, as it makes it mismatched and unlike everything else in the file.

Reply:
Indeed, I'll fix!